sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -275,10 +288,15 @@ export class EventHubClient {
   }
 
   /**
-   * Creates an EventHubProducer that can be used to send events to the Event Hub for which this client
-   * was created.
+   * Creates an Event Hub producer responsible for sending `EventData` to the Event Hub.
+   * Depending on whether a `partitionId` is specified in the `options`, the producer may be created to allow event data to
+   * be automatically routed to an available partition or specific to a partition.
```

Comment:
> If `partitionId` is specified in the `options`, all events sent using the producer will be sent to the specified partition. Otherwise, events are automatically routed to an available partition.

Reply:
Same is applicable to the constructor ref docs for `EventHubProducer` class

Reply:
The constructor docs for `EventHubProducer` won't be generated since the constructor is private, do you want me to update the docs for the private methods as well?

Reply:
Ah, I think you meant the class-level docs for EventHubProducer. Updated there.